variables.tf

```diff
@@ -153,6 +153,18 @@ variable "db_replica_count" {
 
 variable "local_ip" {
   description = "URL used to find user's local IP for use with bastion host"
-  default     = "http://ipv4.icanhazip.com"
+  default     = "http://checkip.amazonaws.com"
   type        = string
 }
+
+variable "engine_version" {
+  description = "Aurora database engine version."
+  type        = string
+  default     = "10.7"
+}
+
+variable "auto_minor_version_upgrade" {
+  description = "Determines whether minor engine upgrades will be performed automatically in the maintenance window"
```

Comment:
Looks good! Can you add something like "...for aurora db" at the end so we know which resource this is for?

Reply:
Sure, I'll modify it.

Reply:
Added comment on `e6ce1af`